pontoon/base/static/js/translate.js

```diff
@@ -168,10 +168,31 @@ var Pontoon = (function (my) {
       var self = this;
       $("#editor")[0].entity = entity;
 
+      /*
+       * Linkifies any traces of URLs present in a given string.
+       * Usage : string.linkify();
+       *
+       * Matches the URL Regex and parses the required matches.
+       * Can find more than one URL in the given string.
+       */
+      if(!String.linkify) {
+        String.prototype.linkify = function() {
+          // http://, https://, ftp://
+          var urlPattern = /\b(?:https?|ftp):\/\/[a-z0-9-+&@#\/%?=~_|!:,.;]*[a-z0-9-+&@#\/%=~_|]/gim;
+          // www. sans http:// or https://
+          var pseudoUrlPattern = /(^|[^\/])(www\.[\S]+(\b|$))/gim;
+
+          return this
+            .replace(urlPattern, '<mark class="placeable"><a href="$&">$&</a></mark>')
+            .replace(pseudoUrlPattern, '<mark class="placeable">$1<a href="http://$2">$2</a></mark>');
+        };
+      }
+
```

Comment:
Could you make this a public function, that takes string as an argument, to match with the rest of Pontoon's code style?